Makefile

```diff
@@ -189,6 +189,10 @@ OPENWEAVE_PROJECT_CONFIG = $(PROJECT_ROOT)/main/include/WeaveProjectConfig.h
 
 OPENTHREAD_PROJECT_CONFIG = $(PROJECT_ROOT)/main/include/OpenThreadConfig.h
 
+OPENTHREAD_CONFIGURE_OPTIONS += \
+    --enable-child-supervision \
+    --enable-jam-detection
```

Comment:
Is it possible to use the new `OPENTHREAD_CONFIG_...` paradigm of using defines in a config.h rather than autotools configure parameters?

Reply:
As @bukepo mentioned in https://github.com/openweave/openweave-core/pull/481#issuecomment-582687182, we'd need to snap to a newer version of OpenThread.